python313/Dockerfile

```
@@ -0,0 +1,51 @@
+FROM python:3.13-slim-bookworm
```

```suggestion
FROM python:3.13.3-slim-bookworm
```
Lets pin the interpreter to avoid surprises updates.